scripts/pre_commit_linter.py

```diff
@@ -162,8 +209,9 @@ def _lint_js_files(node_path, jscs_path, config_jscsrc, files_to_lint):
     jscs_cmd_args = [node_path, jscs_path, config_jscsrc]
 
     for ind, filename in enumerate(files_to_lint):
-        print 'Linting file %d/%d: %s ...' % (
-            ind + 1, num_js_files, filename)
+        if (ind + 1) % 10 == 0:
+            print 'Linting file %d/%d: %s ...' % (
```

Comment:
Perhaps "Linting files 1-10 of 23", "Linting files 11-20 of 23", "Linting files 21-23 of 23"?